deliver/lib/deliver/upload_screenshots.rb

```diff
@@ -134,7 +133,7 @@ def collect_screenshots_for_languages(path, ignore_validation)
       # to have it in there for frameit support
       unaccepted_device_shown = false
       screenshots.select! do |screenshot|
-        exists = Spaceship::Tunes::DeviceType.exists?(screenshot.device_type)
+        exists = !screenshot.device_type.nil?
```

Comment:
I've changed this to just be a simple nil check since `AppScreenshot` has the responsibility of ensuring that the return value of this attribute is valid, otherwise it will just return `nil`